### DISCUSSION TITLE
Hi Dragonmaster

### DISCUSSION THREAD

**Discussion by lesleyespire** [dragon731012/-WORKING-bookmarklets-and-games#24] [General]
Hi, thankyou for all your work, my favourite being the DM-Chat and the Chromebook bookmarklets.
I got a few questions;
How much of these did you solely design?
How did you learn to do this?
Where did you find the ones you didn't design

Thanks again

Pyrobyte

**Comment by dragon731012** [comment]
I made all the ones that say by dragonmaster73101, and I taught myself JavaScript. The ones that I didn’t make, I mostly found on random websites and on GitHub.

**Comment by lesleyespire** [comment]
Okay, thanks
Is there any recommendation you have for learning javascript?
